subprojects/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/DefaultProjectConnection.java

```diff
@@ -75,4 +80,28 @@ public TestLauncher newTestLauncher() {
     public BuildActionExecuter.Builder action() {
         return new DefaultBuildActionExecuter.Builder(connection, parameters);
     }
+
+    @Override
+    public void notifyDaemonsAboutChangedFiles(List<String> locations) {
+        ConsumerOperationParameters.Builder operationParamsBuilder = ConsumerOperationParameters.builder();
+        operationParamsBuilder.setCancellationToken(new DefaultCancellationTokenSource().token());
+        operationParamsBuilder.setParameters(parameters);
+        operationParamsBuilder.setEntryPoint("Invalidate locations API");
```

Comment:
This should use the same terminology as all the rest.

Reply:
Done.